Update JavaScriptCore TBA versions for macOS 13.3 and iOS 16.4

emw-apple opened the pull request:
#### 75ee90767a643d903a09955acaf5e75e62587d47
<pre>
Update JavaScriptCore TBA versions for macOS 13.3 and iOS 16.4
<a href="https://bugs.webkit.org/show_bug.cgi?id=252649">https://bugs.webkit.org/show_bug.cgi?id=252649</a>
rdar://105682433

Reviewed by Patrick Angle and Yusuke Suzuki.

Finalize the availability version numbers for API/SPI that are newly
available in the latest seed, plus some older declarations that were
never updated.

* Source/JavaScriptCore/API/APIIntegrityPrivate.h:
* Source/JavaScriptCore/API/JSContext.h:
* Source/JavaScriptCore/API/JSContextPrivate.h:
* Source/JavaScriptCore/API/JSContextRef.h:
* Source/JavaScriptCore/API/JSContextRefPrivate.h:
* Source/JavaScriptCore/API/JSRemoteInspector.h:

Canonical link: <a href="https://commits.webkit.org/260895@main">https://commits.webkit.org/260895@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/e0c97376d184bf1bcab215231c1c4c9f25668084

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [❌ 🧪 style](https://ews-build.webkit.org/#/builders/6/builds/108820 "2 style errors") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/77/builds/17920 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/43/builds/41653 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/87/builds/319 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/12/builds/118114 "Built successfully") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/76/builds/19376 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/85/builds/9199 "Built successfully") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/36/builds/101050 "Built successfully") | 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/19/builds/114588 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/78/builds/14535 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/3/builds/97748 "Passed tests") | [❌ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/35/builds/42519 "Failed to checkout and rebase branch from PR 10430") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/9/builds/96515 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/73/builds/29387 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/34/builds/84390 "Passed tests") | 
| [✅ 🛠 🧪 jsc](https://ews-build.webkit.org/#/builders/1/builds/97921 "Built successfully and passed tests") | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/81/builds/10699 "Built successfully") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/70/builds/30737 "Passed tests") | [✅ 🛠 jsc-armv7](https://ews-build.webkit.org/#/builders/38/builds/98732 "Built successfully") | 
| [✅ 🛠 🧪 jsc-arm64](https://ews-build.webkit.org/#/builders/83/builds/8827 "Built successfully and passed tests") | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/82/builds/11458 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/84/builds/7671 "Passed tests") | [✅ 🧪 jsc-armv7-tests](https://ews-build.webkit.org/#/builders/46/builds/30988 "Passed tests") | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/80/builds/16845 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/62/builds/50333 "Passed tests") | [✅ 🛠 jsc-mips](https://ews-build.webkit.org/#/builders/37/builds/106349 "Built successfully") | 
| [❌ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/74/builds/7556 "Failed to push commit to Webkit repository") | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/79/builds/13045 "Built successfully") | | [✅ 🧪 jsc-mips-tests](https://ews-build.webkit.org/#/builders/45/builds/26362 "Passed tests") | 
| | | | | 
<!--EWS-Status-Bubble-End-->

patrickangle reviewed (approved):
All the inspection-centric API looks correctly annotated. Someone from JSC should verify the other API/SPI, particularly the ones that appear to have versions older than 13.3/16.4.

Constellation reviewed (approved):
rs=me

emw-apple committed:
Update JavaScriptCore TBA versions for macOS 13.3 and iOS 16.4
https://bugs.webkit.org/show_bug.cgi?id=252649
rdar://105682433

Reviewed by Patrick Angle and Yusuke Suzuki.

Finalize the availability version numbers for API/SPI that are newly
available in the latest seed, plus some older declarations that were
never updated.

* Source/JavaScriptCore/API/APIIntegrityPrivate.h:
* Source/JavaScriptCore/API/JSContext.h:
* Source/JavaScriptCore/API/JSContextPrivate.h:
* Source/JavaScriptCore/API/JSContextRef.h:
* Source/JavaScriptCore/API/JSContextRefPrivate.h:
* Source/JavaScriptCore/API/JSRemoteInspector.h:

Canonical link: https://commits.webkit.org/260895@main
